Makefile

```diff
@@ -249,17 +249,21 @@ check-runbooks:
 # Testing #
 ###########
 
+.PHONY: bench
+bench:
+	go test -run NONE -bench ^Bench -benchmem $(PKGS)
+
 .PHONY: test
 test: test-unit test-rules test-e2e
 
 .PHONY: test-unit
 test-unit:
-	go test -race -short $(PKGS) -count=1
+	go test -run ^Test -race -short $(PKGS) -count=1
 
 .PHONY: test-e2e
 test-e2e: KUBECONFIG?=$(HOME)/.kube/config
 test-e2e:
-	go test -v -timeout=150m ./test/e2e/ --kubeconfig $(KUBECONFIG)
+	go test -run ^Test -v -timeout=150m ./test/e2e/ --kubeconfig $(KUBECONFIG)
```

Comment:
Are the `-run` parameters really needed? I thought `go test` already only runs functions that start with `Test` and `go run -bench .` runs the `Benchmark` prefixed tests.

Reply:
I verified that the tests and benches can indeed be run using `go test` and `go test -bench=.`, not sure why I assumed otherwise. Sorry for the confusion here, reverted.